src/main/java/com/derongan/minecraft/mineinabyss/world/Layer.kt

```diff
@@ -2,6 +2,7 @@ package com.derongan.minecraft.mineinabyss.world
 
 import com.derongan.minecraft.deeperworld.world.section.Section
 import com.derongan.minecraft.mineinabyss.ascension.effect.AscensionEffectBuilder
+import java.awt.geom.FlatteningPathIterator
```

Comment:
Looks like an accidental import of awt stuff

Reply:
fixed